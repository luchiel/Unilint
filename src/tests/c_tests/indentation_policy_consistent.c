int main ()
{
        //hello!
        //I am 8 spaces deep!
        if(1)
                printf("and it's really okay!");
         return 0;
}