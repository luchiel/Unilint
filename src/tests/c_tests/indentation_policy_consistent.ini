[indentation]
indentation_style = spaces
indentation_policy = consistent
