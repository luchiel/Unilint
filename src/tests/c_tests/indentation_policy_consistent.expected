7:1: indent too deep
